src/main/steps/help-with-your-fee/content.ts

```diff
@@ -28,7 +28,8 @@ const en = ({ isDivorce, required, isJointApplication, partner }) => ({
   },
 });
 
-const cy: typeof en = ({ isDivorce, required }) => ({
+const cy: typeof en = ({ isDivorce, required, isJointApplication, partner }) => ({
+  ...en({ isDivorce, required, isJointApplication, partner }),
```

Comment:
now you can delete line2 and any other untranslated empty properties as this will copy it from en object.